packages/apm/src/integrations/tracing.ts

```diff
@@ -493,8 +493,8 @@ export class Tracing implements Integration {
           case 'paint':
           case 'measure':
             const mark = transactionSpan.child({
-              description: `${entry.entryType} ${entry.name}`,
```

Comment:
I'd either coerce both to strings, or none for consistency (using template literals)